src/conf_mode/system-login.py

```diff
@@ -118,18 +121,27 @@ def verify(login):
     if 'radius' in login:
         if 'server' not in login['radius']:
             raise ConfigError('No RADIUS server defined!')
-
+        sum_timeout: int = 0
+        radius_servers_count: int =0
         fail = True
         for server, server_config in dict_search('radius.server', login).items():
             if 'key' not in server_config:
                 raise ConfigError(f'RADIUS server "{server}" requires key!')
-
-            if 'disabled' not in server_config:
+            if 'disable' not in server_config:
+                sum_timeout += int(server_config['timeout'])
+                radius_servers_count += 1
                 fail = False
-                continue
+
         if fail:
             raise ConfigError('All RADIUS servers are disabled')
 
+        if radius_servers_count > MAX_RADIUS_COUNT:
+            raise ConfigError('Number of RADIUS servers more than 25 ')
+
+        if sum_timeout > MAX_RADIUS_TIMEOUT:
+            raise ConfigError('Sum of RADIUS servers timeouts '
+                              'has to be less er eq 50 sec')
```

Comment:
typo in the word `or`

Reply:
Can we probably make this a `vyos.base.Warning()` instead? Else a migration script is needed

Reply:
in 1.3 version if the sum of timeouts is more than 60 sec then login into the system will be discarded. So a warning message is very soft for it. I thought about a migration script. Maybe it would be better to add info in the changelog.